docs/data-loaders.md

```diff
@@ -151,8 +151,8 @@ The data loader implemented above already knows how to handle a list of IDs, and
 ## Using Spring Features such as SsoCallerResolver inside a CompletableFuture
 
 When you write async data fetchers, the code will run on worker threads.
-Spring internally stores some context, for example to make the SsoCallerResolver work, on the thread context however.
-This context wouldn’t be available inside code running on a different thread, which makes features such as SsoCallerResolver not work.
+Spring internally stores some context on the local thread context. 
```

Comment:
I don't think that's really true out of the box. Maybe more generally mention: "Be aware of code that uses `ThreadLocal`, which is a fairly common pattern in WebMVC." ThreadLocal doesn't work when multiple threads are involved, which for example means that ThreadLocal data won't be available in data loaders.

Reply:
Actually this PR needs to be dropped, it is outdated.